packages/darklang/languageTools/parser/expr.dark

```diff
@@ -399,15 +399,50 @@ module Darklang =
 
           match keywordLetNode, identifierNode, symbolEqualsNode, expr, body with
           | Ok keywordLet, Ok identifier, Ok symbolEquals, Ok expr, Ok body ->
-            (WrittenTypes.Expr.ELet(
-              node.range,
-              WrittenTypes.LetPattern.LPVariable(identifier.range, identifier.text),
-              expr,
-              body,
-              keywordLet.range,
-              symbolEquals.range
-            ))
-            |> Stdlib.Result.Result.Ok
+            match identifier.children with
```

Comment:
Funny we've had the `let_pattern` in the grammar for a while, not used by `let`s!
Given the fact that let patterns are used by both `ELet` and `ELambda`, this should probably be extracted into a `LetPattern.parse` (or another name - `parseLetPattern` is probably OK based on your judgment)

Reply:
we had it, I just forgot about it :woman_facepalming: 
Fixed